File: src/main/java/problems/Uri1044_Multiplos.java
package problems;

import java.io.IOException;
import java.util.Scanner;

public class Uri1044_Multiplos {
    public static void main(String[] args) throws IOException {
        Scanner ler = new Scanner(System.in);

        int i = ler.nextInt();
        int j = ler.nextInt();

        if (j % i == 0) {
            System.out.println("Sao Multiplos");

        } else {
            System.out.println("Nao sao Multiplos");
        }
    }
}

File: src/main/kotlin/problems/Uri1044_Multiplos.kt
package problems

import java.util.Scanner

fun main(args: Array<String>) {
    val ler = Scanner(System.`in`)
    val i = ler.nextInt()
    val j = ler.nextInt()
    if (j % i == 0 || i % j == 0)
        println("Sao Multiplos")
    else
        println("Nao sao Multiplos")
}

/*
URI Online Judge | 1044
Múltiplos
Adaptado por Neilor Tonin, URI  Brasil

Timelimit: 1
Leia 2 valores inteiros (A e B). Após, o programa deve mostrar uma mensagem "Sao Multiplos" ou "Nao sao Multiplos", indicando se os valores lidos são múltiplos entre si.

Entrada
A entrada contém valores inteiros.

Saída
A saída deve conter uma das mensagens conforme descrito acima.

Exemplo de Entrada	Exemplo de Saída
6 24

Sao Multiplos

6 25

Nao sao Multiplos
 */